components/brave_wallet_ui/components/desktop/line-chart/custom-tooltip/index.tsx

```diff
@@ -5,53 +5,80 @@
 import * as React from 'react'
 import { TooltipProps } from 'recharts'
 
+// Utils
 import {
-  LabelWrapper,
-  ChartLabel
+  useSafeWalletSelector
+} from '../../../../common/hooks/use-safe-selector'
+import { WalletSelectors } from '../../../../common/selectors'
+import Amount from '../../../../utils/amount'
+
+// Styled Components
+import {
+  TooltipWrapper,
+  ChartBalance,
+  ChartDate
 } from '../style'
 
 type Props = TooltipProps<number, number> & {
-  onUpdateBalance: (value: number | undefined) => void
-  onUpdatePosition: (value: number) => void
+  onUpdateViewBoxHeight: (value: number) => void
 }
 
 function CustomTooltip ({
   active,
   coordinate,
   label,
-  onUpdateBalance,
-  onUpdatePosition,
+  onUpdateViewBoxHeight,
   payload,
   viewBox
 }: Props) {
+  // Selectors
+  const defaultFiatCurrency =
+    useSafeWalletSelector(WalletSelectors.defaultFiatCurrency)
+
+  // Methods
   const parseDate = (date: Date) => {
     const formatedDate = new Date(date).toLocaleDateString('en-US', { month: 'numeric', day: 'numeric', year: 'numeric' })
     const formatedTime = new Date(date).toLocaleTimeString('en-US', { hour: 'numeric', minute: 'numeric', hour12: true })
     return `${formatedDate} ${formatedTime}`
   }
```

Comment:
pref(non-blocking): could we move this function declaration out of the component?

Reply:
Moved to Utils with a test!